### PR TITLE
fix: update packages to address vulnerabilities

### PR DESCRIPTION
**Pull request by JivusAyrus**
## Checklist

- [x] I have discussed my proposed changes in an issue and have received approval to proceed.
- [x] I have followed the coding standards of the project.
- [ ] Tests or benchmarks have been added or updated.
- [ ] Documentation has been updated on [https://github.com/wundergraph/cosmo-docs](https://github.com/wundergraph/cosmo-docs).
- [x] I have read the [Contributors Guide](https://github.com/wundergraph/cosmo/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: admission-server/package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "@hono/node-server": "1.11.0",
     "@wundergraph/cosmo-shared": "workspace:^",
     "dotenv": "^16.4.5",
-    "hono": "4.2.7"
+    "hono": "4.6.12"
```
Comment (StarpTech) [inline admission-server/package.json:15]: Die you test it end to end?

Comment (StarpTech) [reply admission-server/package.json:15]: @JivusAyrus ping

Comment (JivusAyrus) [reply admission-server/package.json:15]: Nope, was still looking at a few things. Will do the test in the end.

Comment (JivusAyrus) [reply admission-server/package.json:15]: Tested cdn and the admission server, it works. The rest is just vitest

**Review by StarpTech — APPROVED** [review]
LGTM

**Pull request merged**